richtextfx/src/main/java/org/fxmisc/richtext/CssProperties.java

```diff
@@ -109,4 +109,33 @@ public String getName() {
             return cssMetaData;
         }
     }
+
+    static class BlinkRateProperty extends StyleableObjectProperty<javafx.util.Duration> {
+        private final Object bean;
+
+        private final CssMetaData<? extends Styleable, javafx.util.Duration> cssMetaData;
+
+        public BlinkRateProperty(Object bean, javafx.util.Duration initialValue) {
+            super(initialValue);
+            this.bean = bean;
+            cssMetaData = new PropertyCssMetaData<>(
+                    this, "-fx-blink-rate",
```

Comment:
What's your opinion? Should "blink-rate" be renamed to "caret-blink-rate" to better specify which object's blink rate is being customized?